src/js/view/popup/scheduleCreationPopup.js

```diff
@@ -254,78 +259,30 @@ ScheduleCreationPopup.prototype._onClickSaveSchedule = function(target) {
     startDate = new TZDate(this.rangePicker.getStartDate()).toLocalTime();
     endDate = new TZDate(this.rangePicker.getEndDate()).toLocalTime();
 
-    if (!title.value) {
-        title.focus();
-
-        return true;
-    }
+    validated = this._validateForm(title, startDate, endDate);
```

Comment:
I think `isValid` or `valid` is enough.

Reply:
How about deleting the variable `valid` and doing `focus()` here?
```js
    if (!this._validateForm(title, startDate, endDate)) {
        if (!title.value) {
            title.focus();
        }

        return false;
    }
```